### ISSUE TITLE
feat: seperate logs for downloading patches and installing patches

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#1470]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [3.9.3](https://github.com/open-wa/wa-automate-nodejs/compare/3.9.2...3.9.3) (2021-03-13)

- 🔊  fix #1469 [`#1469`](https://github.com/open-wa/wa-automate-nodejs/issues/1469)
- Release 3.9.3 [`96d4cca`](https://github.com/open-wa/wa-automate-nodejs/commit/96d4ccab82827466666ff525617a6bc0da4d5a62)
- 🔊  seperated logs for installing and downloading patches #1470 [`12e1e45`](https://github.com/open-wa/wa-automate-nodejs/commit/12e1e45dac4a80e56a0c5b911d3158caca7bc045)
- 🐛  dont show update message when library is already up to date #1468 [`2b88637`](https://github.com/open-wa/wa-automate-nodejs/commit/2b88637b15f3a875019a06478e867fb459485980)
